src/org/exist/storage/BrokerWatchdog.java

```diff
@@ -8,10 +27,19 @@
 
 import org.exist.EXistException;
 
+/**
+ * Traces the lease of a Broker
+ *
+ * Note that tracing the stack is expensive
+ * this should only be done in debug mode by
+ * enabling the System Property {@link #TRACE_BROKERS_PROPERTY_NAME}
```

Comment:
That was me documenting existing behaviour that was undocumented ;-)

I think perhaps you mean we should document the System property? If so, I totally agree.

There are a whole load of System properties that are embedded in eXist code which are undocumented. I discover new ones all the time. So I guess a page in the documentation about System properties would be good - obviously some of them are only relevant to developers.